<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guessedWord</class>
 <widget class="QDialog" name="guessedWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>Translation mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Arial&quot;;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <column>
      <property name="text">
       <string>German</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>French</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>English</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>Article mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Arial&quot;;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <column>
      <property name="text">
       <string>Article</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>German</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>French</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>Plural mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Arial&quot;;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <column>
      <property name="text">
       <string>German Plural form</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>German singular form</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>French singular form</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="textEditorBtn">
     <property name="text">
      <string>Text Editor</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
